schema/deploy/database_functions/graphile_worker_job_definer.sql

```diff
@@ -5,8 +5,17 @@ begin;
 
 create or replace function ggircs_portal_private.graphile_worker_job_definer(task text, payload json)
 returns void as $$
+  declare
+    calls_past_minute int;
+    defer_run_multiplier int;
   begin
-    perform graphile_worker.add_job(task, payload);
+
+    calls_past_minute := (select count(*) from ggircs_portal_private.graphile_worker_timestamp where called_at > (now() - interval '1 minute'));
```

Comment:
why not using the `run_at` column from `graphile_worker.jobs` ?

Reply:
Cool, I wasn't aware of graphile_worker.jobs, it should remove the need for an extra table, I'll try doing it with graphile_worker.jobs